src/app/zones/routes.ts

```diff
@@ -0,0 +1,139 @@
+import { getLastNumberParameter } from '@/router/getLastParameter'
+import type { RouteRecordRaw } from 'vue-router'
+export const actions = (): RouteRecordRaw[] => {
+  return [{
+    path: '-create',
```

Comment:
Hurrah for DNS! (for once 😆 )

Reply:
> Whilst doing the above I noticed that you can no longer have a zone called create-zone. If you tried to view the detail page for a zone called create-zone you would get the zone creation page. Therefore I changed the URL to be /zones/-create. By using an invalid DNS identifier for the path segment we can pretty much guarantee these kinds of things won't happen. There are other options (like maybe create/zone) but this is my preferred right now.

I’m not really sure I’m following this here. How would this not have the exact same problem but for `-create` instead of `create-zone`?

Reply:
The vast majority of our "entity" names need to be DNS compatible. A `-` at the start of a name I believe isn't DNS compatible, which means we won't have an entity with this name.